darwin/utils.py

```diff
@@ -555,6 +580,20 @@ def _parse_darwin_annotation(annotation: Dict[str, Any]) -> Optional[dt.Annotati
         main_annotation.subs.append(dt.make_attributes(annotation["attributes"]))
     if "text" in annotation:
         main_annotation.subs.append(dt.make_text(annotation["text"]["text"]))
+    if "inference" in annotation:
+        main_annotation.subs.append(dt.make_opaque_sub("inference", annotation["inference"]))
+    if "directional_vector" in annotation:
```

Comment:
All those new things won't work, as the dependency functions like `make_opaque_sub` don't exist in this version.